src/providers/keys/client.py

```diff
@@ -40,6 +40,6 @@ def _get_with_blockstamp(self, url: str, blockstamp: BlockStamp, params: Optiona
 
     @lru_cache(maxsize=1)
     @list_of_dataclasses(LidoKey.from_response)
-    def get_all_lido_keys(self, blockstamp: BlockStamp) -> list[LidoKey]:
+    def get_all_lido_keys(self, blockstamp: BlockStamp) -> list[dict]:
```

Comment:
`list_of_dataclasses` affects the return type, the current variant is correct

Reply:
`get_all_lido_keys` and similar functions return mappings, which is expected by the `list_of_dataclasses`. `list_of_dataclasses` is a separate function and its return type is inferred from its own typing.